Added failure example and moved resource location

rajanarun opened the pull request.

rajanarun committed:
keep signal bizness together

rajanarun committed:
Added swift signal failure example

Added swift signal failure example with reason, moved swift signal handle and swift signal to reside next to each other

rajanarun committed:
Update swift-signal-handle.rst

rajanarun committed:
Added data fields to output and script

rs-randallburt commented on swift-signal-handle.rst:
```diff
@@ -143,22 +154,25 @@ Full Example Template
       signal_url:
         value: { get_attr: ['signal_handle', 'curl_cli'] }
         description: Swift signal URL
+      
```
You need to make sure that the callout sections above match these changes to the resulting template.

rajanarun replied:
Good catch, I believe I have fixed this with a commit I just pushed.

rs-randallburt commented:
:-1:  forgot something in our walkthrough. See inline comment.

rajanarun committed:
Synced data field changes to the snippets

Synced data field changes to the snippets and updated wording to educate viewer on this.
